packages/itwin/grouping-mapping-widget/src/widget/components/CalculatedPropertyActionWithVisuals.tsx

```diff
@@ -57,6 +60,17 @@ export const CalculatedPropertyActionWithVisuals = ({
   const [validator, showValidationMessage] = useValidator();
   const [resolvedHiliteIds, setResolvedHiliteIds] = useState<string[]>([]);
   const [colorProperty, setColorProperty] = useState<boolean>(false);
+  const origPropertyName = calculatedProperty?.propertyName ?? "";
+  const [showSaveValidationModal, setShowSaveValidationModal] = useState<boolean>(false);
+  const [invalidCustomCalcs, setInvalidCustomCalcs] = useState<Invalids[]>([]);
+
+  const checkOutliers = async (changedPropName: string): Promise<Invalids[]> => {
+    const accessToken = await getAccessToken();
+    const customCalcProps = await mappingClient.getCustomCalculations(accessToken, iModelId, mappingId, group.id);
+    const changes: Invalids[] = await PropertyValidation({ origPropertyName, changedPropertyName: changedPropName, customCalcProps });
+    setInvalidCustomCalcs(changes);
+    return Promise.resolve(changes);
```

Comment:
This `Promise.resolve` also looks redundant.

Reply:
This was a residual from a previous version.